### ISSUE TITLE
Bug: Wrong icon for changing aspect, when changing wallpaper

### ISSUE THREAD

**Issue by ssssamstrup** [SimpleMobileTools/Simple-Gallery#2526]
Hi

Please consider changing the icon for switching the aspect/portrait/landscape button when changing wallpaper. It currently uses a Fullscreen icon for the button, which is confusing.

In the editor, a suitable icon is used for aspect ratio. :)

Thank you!

Wallpaper changer button with fullscreen icon:
![Screenshot_20220621-071429](https://user-images.githubusercontent.com/4302516/174985301-19baf9ff-443a-4d2f-be2e-ee303ab1a18f.png)

Editor icon:
![Screenshot_20220621-100826](https://user-images.githubusercontent.com/4302516/174984499-4134f724-1c93-4aff-ac0a-7e898a6eb60a.png)

**Comment by tibbi** [comment]
we use 1 icon only as you can toggle between portrait, landscape and square too. You can also pick custom aspect ratio in the latest app version already.

**Comment by ssssamstrup** [comment]
Hi, what I mean is that the icon signals that it means "Fullscreen" (the four arrows), when it should signal "Change aspect/rotation". 

If you replaced the Fullscreen icon with the icon used for Change aspect ratio, it would be a lot clearer.

I used a full 10mins not understanding why there was no way to change it, before I tried pressing the "Fullscreen" icon to see what that was for, and then discovered it changed the aspect/rotation.

Hope you'll reconsider
